packages/android_studio.rb

```diff
@@ -43,7 +43,7 @@ def self.postinstall
     puts 'settings, SDK and tools, execute the following:'.lightblue
     puts 'crew remove android_studio'.lightblue
     puts "rm -rf #{CREW_PREFIX}/.config/Android".lightblue
-    puts "rm -rf #{CREW_PREFIX}/.config/.AndroidStudio3.5".lightblue
```

Comment:
Let `self.remove` do it?

Reply:
This line can be removed as the symlink installed in `CREW_DEST_HOME`

Reply:
Good call.

Reply:
Done.